packages/core/src/controllers/events.ts

```diff
@@ -39,6 +39,32 @@ export class EventClient extends IEventClient {
     );
   }
 
+  public init: IEventClient["init"] = async () => {
+    if (typeof process !== "undefined" && process.env.IS_VITEST === "true") return;
+    try {
+      const initEvent = {
+        eventId: uuidv4(),
+        bundleId: this.core.projectId || "",
```

Comment:
`bundleId` should be used by mobile with app's bundle ID or package Name.
`domain` should be used for web with the current app's domain.

If none is available to you, you can remove both `bundleId` and `domain` and we'll generate one at the API level.